rpcserverhandlers_test.go

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"testing"
+
+	"github.com/decred/dcrd/chaincfg/v3"
+	"github.com/decred/dcrd/dcrjson/v3"
+	"github.com/decred/dcrd/dcrutil/v3"
+	"github.com/decred/dcrd/rpc/jsonrpc/types/v2"
+)
+
+// testCfg holds the cfg for the test server.
+var testCfg = rpcserverConfig{ChainParams: chaincfg.MainNetParams()}
+
+// testServer is a rpcServer stub used for testing handlers.
+var testServer = &rpcServer{cfg: testCfg}
+
+func TestHandleCreateRawTransaction(t *testing.T) {
+	type test struct {
```

Comment:
No need for a separate type and I'd prefer it to also be more consistent with the rest of tests in terms of minimizing indententation like so:

```Go
tests := []struct{
	cmd          *types.CreateRawTransactionCmd
	name, result string
	wantErr      bool
	errCode      dcrjson.RPCErrorCode
}{{
	name: "foo",
	// other fields...
}, {
	name: "foo2",
	// other fields...
}}
```